codetransformer/decompiler.py

```diff
@@ -88,6 +89,52 @@ def _instr(instr, queue, stack, body, context):
     )
 
 
+@_process_instr.register(instrs.POP_JUMP_IF_TRUE)
+@_process_instr.register(instrs.POP_JUMP_IF_FALSE)
+def _process_jump(instr, queue, stack, body, context):
+    stack_effect_until_target = sum(
+        map(
+            op.attrgetter('stack_effect'),
+            takewhile(op.is_not(instr.arg), queue)
+        )
+    )
+    if stack_effect_until_target == 0:
```

Comment:
This is temporary logic anyway.  The other case is ternary/short-circuiting operators.